calico_kubernetes/calico_kubernetes.py

```diff
@@ -265,28 +265,58 @@ def _assign_container_ip(self):
 
         :return IPAddress which has been assigned
         """
+        def _assign(ip):
```

Comment:
Yeah, good point.  I likely this because you didn't end up with multiple `assign` methods like `_assign_container_ip` and `__actually_assign_ip` or something like that, but I can probably make something work.